ck.sh

```diff
@@ -66,10 +66,15 @@ case "$COMMAND" in
 				# Replace with new build files.
 				cp -r dev/builder/release/ckeditor/* ../ckeditor/
 
+				cd ..
+				git add ckeditor
```

Comment:
That's actually much better inmho, I think I might have forgotten to "add the result" once, so :+1: